### ISSUE TITLE
Fields are not updating in contentlayer

### ISSUE THREAD

**Issue by marcofranssen** [contentlayerdev/contentlayer#55]
The author is some static value I retrieve from a different file. (field is not visible in output)
The coverImage I want to get if no value is provided (field is not visible)
The categories and tags is a list of strings (field is also not visible)

See below for the code, frontmatter and the result in contentlayer folder.

**contentlayer.config.js**

```js
const computedFields = {
  readingTime: { type: "json", resolve: (doc) => readingTime(doc.body.raw) },
  wordCount: {
    type: "number",
    resolve: (doc) => doc.body.raw.split(/\s+/gu).length,
  },
  slug: {
    type: "string",
    resolve: (doc) => doc._raw.sourceFileName.replace(/\.md$/, ""),
  },
  author: { type: "json", resolve: (doc) => publisher },
  coverImage: {
    type: "json",
    resolve: (doc) => getCoverImage(doc, defaultData, "coverImage"),
  },
};

const Post = defineDocumentType(() => ({
  name: "Post",
  filePathPattern: `blog/*.md`,
  fields: {
    title: { type: "string", required: true },
    date: { type: "date", required: true },
    tags: {
      type: "list",
      of: { type: "string" },
      typeField: "string",
      required: false,
    },
    categories: {
      type: "list",
      of: { type: "string" },
      typeField: "string",
      required: true,
    },
    coverImage: { type: "string", required: false },
  },
  computedFields,
}));
```

My blogs frontmatter:

```md
---
title: Concurrency in Go
tags:
  - concurrency
  - parallelism
categories:
  - Software development
  - Golang
date: 2019-01-17T17:21:56.000Z
coverImage: /images/be41df8a96b934821f14f5844949b52604c8b074.png
---
```

Result:

```json
{
  "title": "Concurrency in Go",
  "date": "2019-01-17T17:21:56.000Z",
  "coverImage": "/images/be41df8a96b934821f14f5844949b52604c8b074.png",
  "body": {
    "raw": "\nA reason to choose Go over other programming languages could be to have the need for building software which requires concurrency. Go is built with concurrency…"
  },
  "_id": "blog/concurrency-in-go.md",
  "_raw": {
    "sourceFilePath": "blog/concurrency-in-go.md",
    "sourceFileName": "concurrency-in-go.md",
    "sourceFileDir": "blog",
    "bodyType": "markdown",
    "flattenedPath": "blog/concurrency-in-go"
  },
  "type": "Post",
  "readingTime": {
    "text": "9 min read",
    "minutes": 8.22,
    "time": 493200,
    "words": 1644
  },
  "wordCount": 1646,
  "slug": "concurrency-in-go"
}
```

**Comment by schickling** [comment]
Thanks for reporting this issue. Please provide a reproduction repo I can use to investigate this issue. 🙏

**Comment by marcofranssen** [comment]
I have invited you to my repo. Checkout the `contentlayer` branch.

```bash
yarn
yarn dev
```

http://localhost:3000/blog is the first url trying to utilize contentlayer.

**Comment by marcofranssen** [comment]
As also mentioned on the other issue #56 I learned about the cli.

Currently resolved some errors, but still no categories and tags. 

```bash
$ ./node_modules/.bin/contentlayer build --verbose --clearCache
Cache cleared successfully
```

The result of that is I'm only left with the following files.

```bash
.contentlayer
└── .cache
    └── v0.0.32
        ├── compiled-contentlayer-config.mjs
        └── compiled-contentlayer-config.mjs.map

2 directories, 2 files
```

**Comment by marcofranssen** [comment]
This seems to be resolved with the 0.0.33 release.
